src/components/GamePlayConsole.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+interface IProps {
+    shouldHide: boolean;
+}
+
+export default function GamePlayConsole(props: IProps) {
+    return (
+        <section id="gameplay-container" className={props.shouldHide ? "hide" : ""}>
+            {/* tslint:disable-next-line:jsx-no-multiline-js */}
+            {/* <img
+                src={currentEvent.imgPath() === "" ? placeholderImg : currentEvent.imgPath()}
+                alt="Event illustration"
+            /> */}
+            {/* <div id="landing-box"> */}
+            <div 
+              id="gameplay-content-box" 
+              className="nes-container is-centered"
+            >
+                <p>Dear, <span id="enter-name" className="nes-pointer">Enter Name</span>
```

Comment:
This looks like a good work around for now, but we still want to be able to display images right? Either the game console component should be able to take an image as a prop and render it, or an image component should be added elsewhere, since there are now images on the [Google Drive](https://drive.google.com/drive/u/0/folders/1E3f9Vbrq0Rhjg2yK2iMyS4mBOKZjtWau?ths=true) that we should use in our MVP. If put in here, we can probably make use of that boolean prop to, instead of conditionally hide the acceptance letter, decide whether to display the acceptance letter or render the image of the current event. Later on, we could maybe turn the acceptance letter into its own image so we don't have to deal with the boolean at all, but that can be handled later.

Reply:
agreed, we are yet to determine how this is gonna be handled better when more cases are considered. we should have a ticket to address this issue and handled later